tests/secure_verify/host/evidence.h

```diff
@@ -7,7 +7,7 @@
 #include <openenclave/bits/evidence.h>
 #include <openenclave/host.h>
 
-#include "../../../../host/sgx/platformquoteprovider.h"
+#include "../../../host/sgx/platformquoteprovider.h"
```

Comment:
## Why we modify this line
It's falling the pipeline `PR Testing / Ubuntu 1804 Cross Platform Build`

Previously, `../../../../host/sgx/platformquoteprovider.h` only work if build directory is 1 level deep, like `./build/`

> Then why on Windows it builds successfully where the build directory is 2 levels deep, like `./build/debug/`?